include/llpc.h

```diff
@@ -51,6 +51,7 @@
 //* %Version History
 //* | %Version | Change Description                                                                                    |
 //* | -------- | ----------------------------------------------------------------------------------------------------- |
+//* |     39.0 | Added scalarThreshold to PipelineShaderOptions                                                        |
```

Comment:
I think the versioning is changed to just bump minor version (no vesion macro gating) without changing the major one if new fields are added to LLPC interface, not disrupting existing ones.

Reply:
Already changed the version from 39.0 to 38.2 and remove all the version check as well.